index.html

```diff
@@ -4,160 +4,154 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="./Assets/CSS/style.css" />
-    <link rel="stylesheet" href="./Assets/CSS/google.css" />
-    <link rel="stylesheet" href="./Assets/CSS/spotify.css" />
-    <link rel="stylesheet" href="./Assets/CSS/bandsintown.css" />
-    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous" />
-    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyCkb96g-9_ZHg68di_g4cUEDTA5pEi9Ef8">        </script>
```

Comment:
Should there links be removed?

Reply:
probably not

Reply:
can you add them back in and re push?

Reply:
maybe I should just close this pull request since materialize seems the way to go

Reply:
oh you had add the material.io in??? but might just be a matter of swapping out the class names? not sure.

Reply:
yeah originally it was mostly just css positioning. i can always just reopen the branch if necessary too

Reply:
Ok I will close the pull request then.